src/common/src/catalog/mod.rs

```diff
@@ -126,6 +126,57 @@ impl From<TableId> for u32 {
     }
 }
 
+#[derive(Clone, Copy, Debug, Display, Default, Hash, PartialOrd, PartialEq, Eq, Ord)]
+pub struct ColumnIdx {
+    pub column_idx: u32,
```

Comment:
We generally use `usize` for the indices in Rust.

Reply:
I believe after removing `ColumnIdx` I am using `usize` for the indexes 🤔